docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,75 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="Debian_commandline"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by or as root.
+    {:class="alert alert-info"}
+
+    1.  Connect the USB cable to your computer and its mini-B plug to the EV3 (slot marked "PC").
+
+    2.  Verify that USB tethering is disabled: "Wireless and Networks" > "Tethering.
+
+    3.  Set default Linux network settings: "Wireless and Networks" > "All Network Connections" > "Wired" >
```

Comment:
To be consistent with the rest of the website, use italics for name of things printed on the screen. 

    *Wireless and Networks > All Network Connections > Wired*

Also, a screenshot here would be nice. We may have some existing ones that can be reused.